DMTN-036.tex

```diff
@@ -0,0 +1,808 @@
+Exposures with Large CCD Mosaics}
+\author{John Parejko (University of Washington), Pierre Astier (LPNHE/IN2P3/CNRS
+Paris)}
+\setDocRef{DMTN-036}
+\date{\today}
+\setDocAbstract{The jointcal package simultaneously optimizes the astrometric and
+photometric calibrations of a set of astronomical images. In principle
+and often in practice, this approach produces distortion and thoroughput
+models which are more precise than when fitted independently. This
+is especially true when the images are deeper than the astrometric
+reference catalogs. In the ``Astromatic'' software suite, this simultaneous
+astrometry functionality is fulfilled by ``SCAMP''. The code we
+describe here has similar aims, but follows a slightly different route.
+Jointcal is built on top of the the LSST Data Management software
+stack.}
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% User specified LaTeX commands.
+\newcommand{\ClName}[1]{{\bf #1}}
+\newcommand{\RoutineName}[1]{\texttt{#1}}
+%\def\arcsec{\hbox{$^{\prime\prime}$}}
+%\providecommand{\eprint}[1]{\href{http://arxiv.org/abs/#1}{#1}}
+
+% Change history defined here.
+% Order: oldest first.
+% Fields: VERSION, DATE, DESCRIPTION, OWNER NAME.
+% See LPM-51 for version number policy.
+\setDocChangeRecord{%
+  \addtohist{1}{2017-09-01}{Unreleased.}{John Parejko}
+}
+
+\makeatother
+
+\usepackage{xunicode}
+\begin{document}
+\maketitle
+\tableofcontents{}
+
+\section{Introduction}
+
+With deep astronomical images, it is extremely common that the relative
+astrometry and photometry between images is considerably more precise
+than the accuracy of external catalogs, where ``more precise'' can
+be as large as two orders of magnitude. For applications where the
+quality of relative astrometry is important or vital, it is important
+to rely on some sort of simultaneous astrometry solution, if possible
+optimal in a statistical sense.
+
+This package performs a least-squares fit to a set of images. Since
+it aims at statistical optimality, we maximize the likelihood of the
+measurements with respect to all unknown parameters required to describe
+the data. These parameters are mostly in two sets: the position on
+the sky of the objects in common between the images, and the mapping
+of each image to the sky. To these obvious parameters, one can add
+proper motions (where applicable), and parameters describing the differential
+effect of atmospheric refraction on the position of objects. It is
+clear that one cannot fit simultaneously the position on the sky and
+the mappings from CCD coordinates to the sky, without extra constraints:
+the ``sky'' coordinate system is then undefined, and one needs reference
+positions in order to fully define this frame. We use the GAIA catalog
+as our reference catalog, and may supplement it with deeper data (e.g.
+PS1) where available.
+
+SCAMP \footnote{See \href{http://www.astromatic.net/software/scamp}{http://www.astromatic.net/software/scamp}}is
```

Comment:
Cite SCAMP (I really don't like footnote URL citations).

Reply:
What's the preferred reference for SCAMP?

Reply:
https://ui.adsabs.harvard.edu/#abs/2006ASPC..351..112B/abstract